plugin-server/src/worker/ingestion/event-pipeline/runner.ts

```diff
@@ -222,9 +226,16 @@ export class EventPipelineRunner {
             return this.registerLastStep('pluginsProcessEventStep', [event], kafkaAcks)
         }
 
+        // Pass the hogTransformer to transformEventStep
+        const transformedEvent = await this.runStep(
+            transformEventStep,
+            [this.hub.HOG_TRANSFORMATIONS_ALPHA_ENABLED, processedEvent, this.hogTransformer],
```

Comment:
Personally I would way prefer just passing in the hub. That way you can check the config val and pull out the hogTransformer without a bunch of drilling here.